src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+    {!person.isMarried ? (
+      <p className="Person__partner">I am not married</p>
+    ) : (
+      <p className="Person__partner">
+        {person.sex === 'f'
+          ? `${person.partnerName} is my husband`
+          : `${person.partnerName} is my wife`}
+      </p>
+    )}
```

Comment:
```suggestion
   
      <p className="Person__partner">
         {!person.isMarried ? (
           'I am not married'
         ) : (
            person.sex === 'f'
               ? `${person.partnerName} is my husband`
               : `${person.partnerName} is my wife`
            )}
      </p>
```
move condition inside the tag